File: src/main/java/pl/tsimur/jpatest/exception/InvalidArgumentException.java
package pl.tsimur.jpatest.exception;

import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.ResponseStatus;
import org.springframework.web.server.ResponseStatusException;

@ResponseStatus(HttpStatus.BAD_REQUEST)
public class InvalidArgumentException extends ResponseStatusException {

    public InvalidArgumentException(String message) {
        super(HttpStatus.BAD_REQUEST,
                "This value is not correct " + message);

    }
}

File: src/main/kotlin/pl/timur/jpatest/exception/InvalidArgumentException.kt
package pl.timur.jpatest.exception

import org.springframework.http.HttpStatus
import org.springframework.web.bind.annotation.ResponseStatus
import org.springframework.web.server.ResponseStatusException

@ResponseStatus(HttpStatus.BAD_REQUEST)
class InvalidArgumentException(message: String) : ResponseStatusException(
    HttpStatus.BAD_REQUEST,
    "This value is not correct $message"
)